test/spec_rewindable_input.rb

```diff
@@ -167,9 +167,16 @@ def @io.rewind
 
 describe Rack::RewindableInput::Middleware do
   it "wraps rack.input in RewindableInput" do
-    app = proc{|env| [200, {}, [env['rack.input'].class.to_s]]}
-    app.call('rack.input'=>StringIO.new(''))[2].must_equal ['StringIO']
+    app = proc { |env| [200, {}, [env['rack.input'].class.to_s]] }
+    app.call('rack.input' => StringIO.new(''))[2].must_equal ['StringIO']
     app = Rack::RewindableInput::Middleware.new(app)
-    app.call('rack.input'=>StringIO.new(''))[2].must_equal ['Rack::RewindableInput']
+    app.call('rack.input' => StringIO.new(''))[2].must_equal ['Rack::RewindableInput']
```

Comment:
Happy to undo these changes if desired and potentially also drop the spaces in the new test. It looked like newer code preferred this syntax.

Reply:
For the sake of back porting and review, please don't include unrelated changes.

Reply:
@ioquatix, want me to drop spaces in the new test for consistency inside this file then?

Reply:
I removed the spaces. It does add a couple new RuboCop offenses so let me know if you prefer it to be different.